docs/src/main/sphinx/installation/cli.rst

```diff
@@ -188,6 +188,12 @@ mode:
       operating system username. You can override the default username,
       if your cluster uses a different username or authentication mechanism.
 
+Most of the options can also be set as parameters in the URL. This means
+a JDBC URL can be used in the CLI after removing the ``jdbc:`` prefix.
+However, the same parameter may not be specified using both methods.
+See :doc:`the JDBC driver parameter reference </installation/jdbc>`
+to find out URL parameter names.
```

Comment:
this will need an example or two..

Reply:
Added. I also added a note saying that the password parameter cannot be set in the URL, PTAL.